apps/src/templates/Notification.jsx

```diff
@@ -145,7 +171,7 @@ const Notification = ({
               )}
             </div>
           </div>
-          <div style={desktop ? null : styles.buttonsMobile}>
+          <div style={desktop ? buttonsStyles : styles.buttonsMobile}>
```

Comment:
1) Maybe call this `buttonsStylesDesktop` if it's only applied in the desktop context,
2) Does the need to pass custom styles only apply to desktop or do we need to apply them in a mobile context too?

Reply:
1. see 2
2. Will add to both. This was just an oversight